adjust_dataset.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+import argparse
+
+HAPPINESS = 3
+
+
+def main():
+    """
+    Run this script to create new csv file with dataset for Smile-Warrior from fer2013.csv dataset for Facial
+    Expression Recognition.
+
+    This script is changing labels of emotions in fer2013.csv to 1 - smile, 0 - no smile.
+
+    To run correctly you need to pass two paths to csv files as a command-line arguments, first for input file which
+    will be adjusted for Smile-Warrior needs and second for output file where new dataset will be created.
+
+    Example:
+
+        $ python adjust_dataset.py -input fer2013.csv -output smile_warrior_dataset.csv
```

Comment:
Not sure if needed. Argparser already gives help on how to use it.

Reply:
Ok, I removed it.